### PR TITLE
fix(client): remove duplicated DMN tutorial entry Closes #3032

### LINKED ISSUE #3032: Help menu contains a duplicate entry to 'DMN Tutorial'
### Describe the bug

image

Two entries are present in the menu for 'DMN Tutorial', they both point to the same link. We should clean this up as it doesn't serve any purpose.

### Steps to reproduce

1. Open the modeler
2. Navigate to menu > help

### Expected behavior

There should only be one entry. 

### Environment

- Camunda Modeler Version: 5.1

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by pawel975**
Closes #3032

### CODE REVIEW

File: app/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,1122 @@
 {
   "name": "camunda-modeler",
   "version": "5.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (pinussilvestrus) [inline app/package-lock.json:4]: Let's not push these changes and keep lock file version 1. This pull request should not need any lock file changes.

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution 👍 The change looks good from my perspective. Three things we need to consider before merging this.

* Can you update the commit to be aligned with our commit history? Something like

```sh
fix(client): remove duplicated DMN tutorial entry
Closes #3032
```

* Which npm version are you using? The PR includes a lot of unrelated lock file changes that shall be removed.
* As the PR includes a fix, please direct it to `master` instead of `develop`.

Let me know if you need help with something 👍

File: app/package-lock.json (hunk 1 above)
Comment (pawel975) [reply app/package-lock.json:4]: Hi, my npm version is 8.1.2. I'll try to create new pull request and follow your remarks 👍

**Comment by pawel975** [comment]
@pinussilvestrus I created new PR with your requested changes in #3039 but frankly speaking I don't know what's blocking that PR.

**Comment by pinussilvestrus** [comment]
Hi @pawel975, seems like the other PR was originally branched from `develop`; that's why you see these other changes. 

I will wrap it up and include your bug-fixing changes as proposed. 👍

**Comment by pinussilvestrus** [comment]
I incoorporated your changes via https://github.com/camunda/camunda-modeler/pull/3044. Thanks again 👍

**Pull request closed without merging**